src/App.vue

```diff
@@ -20,13 +20,10 @@
       </v-fade-transition>
 
       <!-- SBC Common Components header -->
-      <sbc-header
+      <SbcHeader
         class="sbc-header"
-        :in-auth="false"
-        :redirectOnLoginSuccess="nameRequestUrl"
-        :redirect-on-login-fail="nameRequestUrl"
-        :redirect-on-logout="nameRequestUrl"
```

Comment:
These just got passed back to us when the header login/logout actions redirect back our signin/signout routes. So I deleted them and just handle the URLs on the Signin and Signout pages.